server/constants/constants.go

```diff
@@ -104,3 +104,18 @@ const (
 	GitBranchIcon = "git-branch-icon.svg"
 	ProjectIcon   = "project-icon.svg"
 )
+
+var ValidSubscriptionEventsForBoards = map[string]bool{
```

Comment:
```suggestion
var (
    ValidSubscriptionEventsForBoards = map[string]bool{
```
You can create a block for all the vars.